microk8s-resources/actions/disable.dns.sh

```diff
@@ -6,7 +6,8 @@ set -e
 skip_opt_in_config() {
     opt="--$1"
     config_file="$SNAP_DATA/args/$2"
-    sudo gawk -i inplace '!/^'"$opt"'/ {print $0}' "${config_file}"
+    sudo AWKLIBPATH="${SNAP}/usr/lib/x86_64-linux-gnu/gawk" AWKPATH="${SNAP}/usr/share/awk/" \
```

Comment:
Heard a rumor that sed can eat the newline too.

Reply:
Fixed :)